README.md

```diff
@@ -1,27 +1,37 @@
 # Contenta
 
-## Quick Install
+## Requirements
 
 - Install [composer](https://getcomposer.org/)
 - Make sure you have the sqlite extension for PHP (if you're using the default install on a mac, this should already be there)
 `sudo apt-get install php-sqlite3`
-- Run the following:
+
+## Quick Install
+
 ```bash
 php -r "readfile('https://raw.githubusercontent.com/contentacms/contenta_jsonapi/8.x-1.x/installer.sh');" > contentacms.sh
 chmod a+x contentacms.sh
 ./contentacms.sh
 ```
 
+- To restart the webserver, run `composer run-script start-contenta <install location>`
 - Visit [http://127.0.0.1:8888/](http://127.0.0.1:8888/) and log into your site with `admin`/`test`
+- The host and port can be overridden by copying `.env` to `.env.local`
```

Comment:
I had to take this out :(
It only works for the quickstart, so there isn't an opportunity to edit it